vallidations/postsValidation.js

```diff
@@ -40,12 +40,31 @@ const patchSchema = {
     content: {
       type: "string",
       minLength: 1,
-    }
+    },
   },
-  additionalProperties: false
-}
+  additionalProperties: false,
+};
+const commentSchema = {
+  type: "object",
+  required: ["comment", "creation_date"],
```

Comment:
I think it will be better to require `post_id` too

Reply:
here we use post_id via req.params.id instead of using it from the body and the goal is that the id does not come through the body, therefore I think this is a better option